content/getting-started/canary.md

```diff
@@ -194,7 +194,7 @@ kubectl delete namespace $NAMESPACE
 
 ### Inspect progress using Grafana
 
-Coming soon
+You can inspect the progress of your experiment using the sample *iter8 Metrics* dashboard. To install this dashboard, see [here](../integrations/grafana/).
```

Comment:
This link does not work

Reply:
Should it be

```
[here](../../integrations/grafana/)
```

Reply:
Here is the deploy preview link: https://deploy-preview-210--focused-benz-34dc64.netlify.app/getting-started/canary/

Reply:
@Alan-Cha thanks for catching; fixed. How do I figure out the preview link?